components/Form/index.js

```diff
@@ -0,0 +1,328 @@
+import styled from "styled-components";
+
+export default function Form({
+  handleAddPlant,
+  plant,
+  buttonText,
+  handleToggleModal,
+  handleEditPlant,
+}) {
+  function handleSubmitAddPlant(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    const selectedSeasons = formData.getAll("fertiliserSeason");
+    if (selectedSeasons.length === 0) {
+      alert("Please select at least one season.");
+      return;
+    }
+    data.fertiliserSeason = selectedSeasons;
+
+    { buttonText === "Edit" && handleEditPlant(data, plant.id) }
+
+    { buttonText === "Add" && handleAddPlant(data) }
```

Comment:
The logical `&&` operator is a great standalone solution, but since you have an if/else scenario here, I would prefer to write a classic if/else statement like you already did it in the _app.js because it makes the code much easier to read.

Reply:
Thanks, we changed it, and it makes more sense, you're right